app/src/styles/_cssReset.scss

```diff
@@ -0,0 +1,49 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
In this piece of code you could use this approach to reset the values:
* {
 //Elements that we want to reset
}